aspnetcore/fundamentals/servers/yarp/packages-refs.md

```diff
@@ -0,0 +1,128 @@
+# Packages referenced by YARP
```

Comment:
This doesn't seem like it needs to be in these docs. We can continue to keep it in the repo somewhere.